tifeatures/factory.py

```diff
@@ -1,29 +1,29 @@
 """tifeatures.factory: router factories."""
 
 import csv
-import json
 from dataclasses import dataclass, field
 from typing import Any, Callable, Dict, Generator, Iterable, List, Optional
 
 import jinja2
+import orjson
```

Comment:
`orjson` is an optional dependency 

```python
try:
    import orjson as json

except ImportError:  # pragma: nocover
   import json

```

Reply:
They're not really interchangeable unless we make some kind of wrapper since orjson returns bytes and json returns str.

Reply:
yes :-(